docs/measurement/measurement_registration.md

```diff
@@ -0,0 +1,123 @@
+# Measurement Registration
+
+To register measurements, start by [navigating](measurement_introduction.md#measurement-navigation) into the measurement summary view.
+![measurement_summary](images/measurement_summary_no_measurements.png)
+Once within the measurement summary view, measurements can be registered via the following steps:
+
+1. [Download](#download-template) the domain specific metadata template spreadsheet
+2. [Prepare](#prepare-metadata) the downloaded metadata sheet with the domain specific mandatory information
+3. [Upload](#measurement-upload) the filled in measurement metadata sheet
+
+!!! info "Project role"
+    Should you not see the registration and download buttons,
+    please make sure that you have been granted the "write" or "admin" role to it by the project owner/admin!
+
+## Download Template
+
+### Proteomics
+
+Within this view download the proteomic specific templates _.xlsx_ file via the template component on the top right by clicking
+on the download icon(down arrow).
+![register_measurements_proteomics_download_template.png](images/measurement_registration_proteomics_download_template.png)
+
+### Genomics
+
+Within this view download the genomic specific templates xlsx file via the template component on the top right by clicking
+on the download icon(down arrow).
+
+Once downloaded open the template file in Microsoft Excel, which contains the two sheets "Property Information" and "Metadata".
+![register_measurements_ngs_measurement_template.png](images/measurement_registration_ngs_measurement_template.png)
```

Comment:
actually, this whole part is a bit superfluous, as you explain it in more detail for each of the templates. Maybe just state: "The templates can be opened with Microsoft Excel..."?